lib/libpcsc-cpp/src/SmartCard.cpp

```diff
@@ -114,6 +134,11 @@ class CardImpl
 
     bool readerHasPinPad() const
     {
+        // Some readers claim to have PinPAD support even if they have not
+        // HID Global OMNIKEY 3x21 Smart Card Reader / HID Global OMNIKEY 6121 Smart Card Reader
+        if ((id_vendor == 0x076B && id_product == 0x3031)
+            || (id_vendor == 0x076B && id_product == 0x6632))
```

Comment:
Using named `constexpr` constants adds to code readability and the condition can be simplified:
```suggestion
        if (id_vendor == VENDOR_HID_GLOBAL &&
            (id_product == OMNIKEY_3x21 || id_product == OMNIKEY_6121))
```

Reply:
This violates C++ Core Guidelines, [ES.45: Avoid “magic constants”; use symbolic constants](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-magic), but I'm OK with this.

What about simplifying the condtion?

Reply:
Feel free to adjust to your liking